docs/platform/server-v3/modules/about.md

```diff
@@ -14,100 +14,132 @@ enterprise_only: true
+
+- [Falcon Module](./module-api) - This should be a structured representation of module definition. Only one can exist per module.
 
-Falcon Module is a package that contains implementations of all the pieces required to achieve particular thing. It can contain none, one or many of these:
-- [Data Source](../data-sources) (used by GraphQL resolvers to fetch the data from various services)
-- [Event Handler](../event-handlers)
-- [Rest Endpoint Handler](../rest-endpoints) (webhook handler)
-- anything else that's needed during request handling
+### Old vs new approach
 
-So in comparison to Falcon Server 2 it's a container that groups all the above things as one package.
+In previous versions of Falcon Server (version 2), there was a distinct separation between extensions, API clients, rest endpoint handlers, and event handlers. Each of these needed to be implemented and loaded as a separate package. Falcon Server 3 changes that approach and introduces the concept of a module.
+
+So in comparison to version 2, version 3 modules are a container that groups all the above things into one package.
+
+In Falcon Server v2 it was necessary to add multiple packages to have full coverage of features. If we consider shop features that will be handled service X (Magento, BigCommerce, CommerceTools, etc) the following packages are required:
 
-In Falcon Server 2 it was necessary to add multiple packages to have full coverage of features. If we consider shop features that will be handled service X (Magento, BigCommerce, CommerceTools etc) the following packages ere required:
 - `@deity/falcon-shop-extension` that provides GraphQL schema for shop features
 - `@deity/falcon-X-api` that provides resolvers and all the calls to service X (implements the shop features)
 - `@deity/falcon-X-endpoints` that provides REST handlers (e.g. webhooks, or callbacks for payments)
 - `@deity/falcon-X-event-handlers` that provides handlers for events in Falcon Server (if such are required)
 
-In Falcon Server you need only:
-- `@deity/falcon-shop-extension` that provides GraphQL schema for shop features
-- `@deity/falcon-X-module` that provides implementation of all the required things in one package
+In Falcon Server v3 you need only:
+
+- `@deity/falcon-X-extension` that provides GraphQL schema for a specified feature e.g. shop or blog
+- `@deity/falcon-X-module` that provides an implementation of all the required things in one package
 
 ## Module and extension relation
 
-Please keep in mind that extension packages are still needed, as extension provide GraphQL schema that will be exposed to GraphQL clients. 
-So now, the 2 things need to be provided:
+Please keep in mind that extension packages are still needed, as extensions provide GraphQL schema that will be exposed to GraphQL clients.
```

Comment:
```suggestion
Please keep in mind that extension packages are required because they provide GraphQL schema that will be exposed towards GraphQL clients.
```

Reply:
done